src/pages/profile.tsx

```diff
@@ -44,14 +63,48 @@ export default function ProfilePage() {
   }
 
   return (
-    <div>
+    <div className="ProfilePage">
```

Comment:
Where is this className used?

Reply:
I think nowhere yet, I simply renamed the class that was copied from the balance page

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/45410089/190174992-516eab09-3b5e-430c-87eb-a57840c4541c.png">

Reply:
It was used in balance.css but looks like we can remove it.

Reply:
Yes, we can remove this 👍